### ISSUE TITLE
Notehead size

### ISSUE THREAD

**Issue by amitgur** [0xfe/vexflow#564]
Any way I can change the size of the note heads?

**Comment by gristow** [comment]
There is not an easy way to size noteheads right now -- though certainly for things like [cue notes](https://en.wikipedia.org/wiki/Cue_note) it would be very useful. (Technically, I believe it is not just the noteheads but in fact also beams & stems whose proportion is meant to be reduced in cue notes. This reminds me of Cyril's comment from #407 about _ossias_, that VexFlow doesn't yet support them well.)

TLDR: there's not an easy way to do this now; there should be. I'm flagging this as a feature request. It may be a while before anyone gets to it since it's still somewhat of an edge case compared to other features in the works.

**Comment by deemaagog** [comment]
Hi, any updates on this? Is it currently possible to draw cue notes?

**Comment by sschmidTU** [comment]
it is. you can even set the scale.

this is how we do it in OSMD:
```typescript
if (firstNote.IsCueNote) {
    vfnoteStruct.glyph_font_scale = Vex.Flow.DEFAULT_NOTATION_FONT_SCALE * Vex.Flow.GraceNote.SCALE;
    vfnoteStruct.stroke_px = Vex.Flow.GraceNote.LEDGER_LINE_OFFSET;
}

if (gve.parentVoiceEntry.IsGrace || gve.notes[0].sourceNote.IsCueNote) {
    vfnote = new Vex.Flow.GraceNote(vfnoteStruct);
```
